lib/Cake/Test/Case/Model/ModelWriteTest.php

```diff
@@ -293,6 +313,27 @@ public function testSaveUuidNull() {
 	}
 
 /**
+ * Ensure that if the id key is null but present the save doesn't fail (with an
+ * x sql error: "Column id specified twice")
+ *
+ * @return void
+ */
+	public function testSaveUuidNullNative() {
+		$this->skipIf(!($this->db instanceof Postgres), 'This test is compatible with Postgres only.');
```

Comment:
This model layer tests. So it require fixtures loading that done in this layer only